:book:  Updated readme with how to create standalone controller-gen binary (#537)
kubernetes-sigs/controller-tools: README.md

of your `GOPATH`. However, if you'd like to continue to work from within your `GOPATH`, please
export `GO111MODULE=on`.

> 

Reviewer: WDYT about add `## Install`

> If just the controller-gen binary is required then run-

Reviewer: ```suggestion
To install only the controller-gen binary :
```

>     go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1

Reviewer: ```suggestion
    ```sh
    go get sigs.k8s.io/controller-tools/cmd/controller-gen@<tag release version>
    ```
```

> Please export `GO111MODULE=on` and the binary would be installed at `$GOPATH/bin`, 
>  see also [`#520`](https://github.com/kubernetes-sigs/controller-tools/issues/520).

Reviewer: ```suggestion
**NOTE** Please export `GO111MODULE=on` and the binary would be installed at `$GOPATH/bin`.
```
